code/modules/spells/spell_types/pointed/coinflip.dm

```diff
@@ -0,0 +1,63 @@
+/datum/action/cooldown/spell/pointed/coinflip
+
+	name = "Fear And Hunger"
+	desc = "This spell causes the target to flip an imaginary coin, and if they call it wrong they lose a random limb"
+	button_icon_state = "autotomy"
+
+	/// Message showing to the spell owner upon activating pointed spell.
+	active_msg = "You prepare a spectral coin."
+	/// Message showing to the spell owner upon deactivating pointed spell.
+	deactive_msg = "You put the coin back in your pocket..."
+	/// The casting range of our spell
+	cast_range = 5
+	/// Variable dictating if the spell will use turf based aim assist
+	aim_assist = TRUE
+
+	sound = 'sound/items/coinflip.ogg'
+	school = SCHOOL_CONJURATION
+	cooldown_time = 120 SECONDS
+
+	invocation = "F'R ND H'NGR"
+	invocation_type = INVOCATION_WHISPER
+	spell_requirements = SPELL_REQUIRES_NO_ANTIMAGIC
+
+/datum/action/cooldown/spell/pointed/coinflip/is_valid_target(atom/cast_on)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!ishuman(cast_on))
+		return FALSE
+
+	return
+
+/datum/action/cooldown/spell/pointed/coinflip/cast(mob/living/carbon/human/cast_on)
+	. = ..()
+	if(cast_on.can_block_magic(antimagic_flags))
+		to_chat(cast_on, span_notice("You hear a coin flipping in the distance, but nothing more."))
+		to_chat(owner, span_warning("The spell had no effect!"))
+		return FALSE
+
+	to_chat(cast_on, span_warning("Heads, or tails!"))
+	cast_on.SetImmobilized(10 HOURS)
+	select_coin(cast_on)
+	
+	return TRUE
+
+
+/datum/action/cooldown/spell/pointed/coinflip/proc/select_coin(mob/living/user, title = "Heads, or Tails?")
+	var/answerChoice = tgui_input_list(user, "Heads or tails", "Call it", list("heads","tails"))
```

Comment:
This has no time limit so you can theoretically keep this open forever.

Reply:
yeah I was thinking about how to get around that, is there a way to time it so if they take too long it'll close and choose for them?

Reply:
``timeout`` arg in tgui_input_list, if you're not moving to something like radial.